hal/halx86/smp/amd64/halmacpi.rc

```diff
@@ -0,0 +1,5 @@
+#define REACTOS_VERSION_DLL
+#define REACTOS_STR_FILE_DESCRIPTION  "X64 Multiprocessor Hardware Abstraction Layer"
```

Comment:
Could both files be merged, with an `#if ...`?
Where is this file(s) used, anyway?

Reply:
I'm working on both 32bit/64bit halmacpi at the same time. I will however change this into being one file as that does make much more sense! Thank you